homeassistant/components/wmspro/generic_entity.py

```diff
@@ -0,0 +1,51 @@
+"""Generic entity for the WMS WebControl pro API integration."""
+
+from __future__ import annotations
+
+from wmspro.destination import Destination
+
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity import Entity
+
+from .const import ATTRIBUTION, DOMAIN, MANUFACTURER
+
+
+class WebControlProGenericEntity(Entity):
+    """Foundation of all WMS based entities."""
+
+    _attr_attribution = ATTRIBUTION
+    _attr_has_entity_name = True
+    _attr_name = None
+
+    def __init__(self, dest: Destination) -> None:
+        """Initialize the entity with destination channel."""
+        super().__init__()
```

Comment:
```suggestion
```

Reply:
Why is this not needed? Isn't it best practice to call the parent objects init method in case of necessary initialization there?

Reply:
There is no parent constructor :)